### DISCUSSION TITLE
brew install is not working -showing curl(35) error

### DISCUSSION THREAD

**Discussion by shivak3** [Homebrew/discussions#3412] [Linux]
Hi,

I created a "linuxbrew" account and configured git and curl from source on CentOS 7.

"brew install" command is showing the curl related error

~~~
==> Downloading https://ghcr.ip/v2/homebrew/core/m4/manifests/1.4.19
curl: (35) OpenSSL SSL_connect: SSL_ERROR_SYSCALL in connection to ghrc.io:443

Error: fortune Failed to download resource "m4_bottle_manifest"
Download failed: https://ghcr.ip/v2/homebrew/core/m4/manifests/1.4.19
~~~

Could you please guide me to fix this error? Thank you in advance.

**Comment by chaowu2009** [comment]
got the same error. could not find help anywhere.

**Reply by rogeriosantosluz** [reply]
same error here:

$ brew install unixodbc

Running `brew update --auto-update`...
==> Fetching dependencies for unixodbc: m4 and libtool
==> Fetching m4
==> Downloading https://ghcr.io/v2/homebrew/core/m4/manifests/1.4.19
curl: (22) The requested URL returned error: 503                              
Warning: Problem : HTTP error. Will retry in 1 seconds. 3 retries left.
curl: (22) The requested URL returned error: 503                              
Warning: Problem : HTTP error. Will retry in 2 seconds. 2 retries left.
curl: (22) The requested URL returned error: 503                              
Warning: Problem : HTTP error. Will retry in 4 seconds. 1 retries left.
curl: (22) The requested URL returned error: 503                              

Error: unixodbc: Failed to download resource "m4_bottle_manifest"
Download failed: https://ghcr.io/v2/homebrew/core/m4/manifests/1.4.19

**Reply by SMillerDev** [reply]
I saw some GitHub actions outages yesterday. Does it work today?

**Comment by srobadelic** [comment]
I am having a slightly different error on a newly imaged machine during the front-page command-line install..
==> /usr/bin/sudo /usr/sbin/chown -R XXXXX:admin /usr/local/Homebrew
==> Downloading and installing Homebrew...
fatal: '...' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
Failed during: git fetch --force origin

**Comment by rogeriocarvalhopacheco** [comment]
Got the same error ...
